servlet-feature-pack/feature-pack-build.xml

```diff
@@ -45,6 +45,9 @@
                 subsystems="configuration/host/subsystems.xml"
                 output-file="domain/configuration/host-slave.xml" />
     </config>
+    <copy-artifacts>
+        <copy-artifact artifact="javax.validation:validation-api:::${version.javax.validation.ee7}" to-location="modules/system/layers/base/javax/validation/api/main/"/>
```

Comment:
This will break if we don't upgrade the build tools. However the upgrade to the build tools will break any hard-coded versions in a `module.xml`.

Reply:
@jamezp yes, this is exactly why I need the build tools upgrade.